src/main/java/frc/robot/Robot.java

```diff
@@ -194,7 +194,8 @@
    */
   @Override
   public void robotInit() {
-    setCompressor();
+    // Set the compressor in closed loop control to enable it.
```

Comment:
Perhaps "enable" should be changed to "configure" in this comment because the `setClosedLoopControl` method is used for compressor configuration.

Reply:
I have kept it as "set" because that is the language used in the [WPILib documentation](https://first.wpi.edu/FRC/roborio/release/docs/java/edu/wpi/first/wpilibj/Compressor.html#setClosedLoopControl(boolean)).